samba/config.json

```diff
@@ -30,6 +30,13 @@
       "10.0.0.0/8",
       "172.16.0.0/12",
       "192.168.0.0/16"
+    ],
+    "veto_files": [
+      "._*",
```

Comment:
._* is for MacOS that writes a ._filename file for every other file on the filesystem. "Dot underscore files are Mac system files and unrelated to Dropbox. They are the result of storing your files on a hard drive partition that does not support extended file attributes."

icon? is "the implementation of "Custom user-icon" in MacOS. If user elects to paste any image of his liking to serve as the icon of some file or directory - it will end up as a hidden iCon?"

I've trimmed down the list here to the most commonly found files from the Samba config for MacOS server that Apple blocks "/icon?/TheVolumeSettingsFolder/.DS_Store/.TemporaryItems/TheFindByContentFolder/.FBCindex/.FBCIndexCopy/.localized/Network Trash Folder/Temporary Items/Cleanup At Startup/.FBCLockFolder/.Trashes/._*/"